lib/record_store/provider/cloudflare.rb

```diff
@@ -123,7 +127,12 @@ def build_from_api(api_record)
         when 'A', 'AAAA'
           record.merge!(address: api_record['content'])
         when 'CNAME'
-          record.merge!(cname: api_record['content'])
+          if api_record['settings']['flatten_cname'] == false
```

Comment:
as per my above comment we should try to be consistent in our sue of symbol or string to index the hash.. should this be `:settings` instead of `'settings'`?

Reply:
others like `'content'` below too